src/gsi/gsi_obOperTypeManager.F90

```diff
@@ -280,20 +286,18 @@ function dtype2index_(dtype) result(index_)
     case("ompstc8"); index_= iobOper_oz
     case("ompsnp" ); index_= iobOper_oz
     case("ompsnm" ); index_= iobOper_oz
-    case("omieff"   ); index_= iobOper_oz
-    case("tomseff"  ); index_= iobOper_oz
-    case("ompsnmeff"); index_= iobOper_oz
 
   case("o3l"    ,"[o3loper]"    ); index_= iobOper_o3l
     case("o3lev"    ); index_= iobOper_o3l
     case("mls20"    ); index_= iobOper_o3l
     case("mls22"    ); index_= iobOper_o3l
     case("mls30"    ); index_= iobOper_o3l
     case("mls55"    ); index_= iobOper_o3l
-    case("ompslp"   ); index_= iobOper_o3l
+    case("omieff"   ); index_= iobOper_o3l
```

Comment:
Why are there changes in the ozone section of this file?

Reply:
Updated this section to match the Develop branch.

Reply:
See comment above

Reply:
Corrected "oz" and "o3loper" blocks to match develop branch